roles/dbid-logger/tasks/main.yml

```diff
@@ -17,7 +17,9 @@
   ansible.builtin.shell: |
     set -o pipefail
     sqlplus -s / as sysdba <<EOF
-    SELECT dbid FROM v\$database;
+    SET HEADING OFF
```

Comment:
Suggested wrapper text (internal link):  http://shortn/_ojm168852N

Notably, right now we ignore SQL errors.  We could add a "whenever sqlerror" clause, but do we actually want DB creation to fail if we couldn't get the DBID?  Perhaps combining whenever sqlerror with ignore_errors would propagate the failure in structured logging.